firstQuestion.js

```diff
@@ -0,0 +1,43 @@
+const set1= new Array(3,1,7,9);
+const set2= new Array(2,4,1,9,3);
+let Distinct= new Array();
+let temp;
+let found= false;
+for( let i=0; i<set1.length ; i++){
```

Comment:
Try to format your code to follow JS conventions (like spacing around operators, around brackets, etc.), which would make this line:

```js
for (let i = 0; i < set1.length; i++) {
```

Also read more about JavaScript's conventions [here](http://udacity.github.io/frontend-nanodegree-styleguide/javascript.html).